backend/lib/endpoints/posts.js

```diff
@@ -157,12 +157,10 @@ router.delete(
   passport.authenticate("jwt", { session: false }),
   async (req, res) => {
     try {
-      const removedPost = await Post.findOneAndRemove({
-        _id: req.params.postId,
-      });
-      res.status(200).json({ sucessMessage: "Successfully deleted post" });
+      const removedPost = await Post.findByIdAndRemove(req.params.postId);
+      res.status(200).json(removedPost);
     } catch (error) {
-      res.status(404).send(error);
+      res.send(error); // not sure why error is {} here
```

Comment:
yeah if something goes wrong you probably want to send all the error info to the client
but that's why we want to change the web framework and take a more systematic approach to error handling, logging, API responses etc.